# PlantPhylomojiChromEvol
PlantPhylomojiChromEvol

This is based on the orginal idea of G Huerta Ramos https://github.com/ghuertaramos/PlantPhylomoji

These are some of the packages that has been used here:

Cayuela L, la Cerda ÍG, Albuquerque FS & Golicher DJ. (2012). taxonstand: An r package for species names standardisation in veg-etation databases. Methods  in  Ecology  and  Evolution, 3, 1078–1083. https   ://doi.org/10.1111/j.2041-210X.2012.00232.x

Jin Y & Qian H. (2019). V.PhyloMaker: an R package that can generate very large phylogenies for vascular plants. – Ecography 42: 1353–1359 (ver. 0).

Wickham H (2016). ggplot2: Elegant Graphics for Data Analysis. Springer-Verlag New York. ISBN 978-3-319-24277-4, https://ggplot2.tidyverse.org.

Yu G, Smith DK, Zhu H, Guan Y & Lam V. (2017). ggtree: an R package for visualization and annotation of phylogenetic trees with their covariates and other associated data. Methods in Ecology and Evolution. 8(1):28-36. doi: 10.1111/2041-210X.12628.

Yu G & Ekstrøm CT. (2019). emojifont: Emoji and Font Awesome in Graphics ver. 0.5.3 https://guangchuangyu.github.io/emojifont 


These are some paper to have into account for the genome size and chromosome number:

Argout, X., Salse, J., Aury, J. M., Guiltinan, M. J., Droc, G., Gouzy, J., ... & Abrouk, M. (2011). The genome of Theobroma cacao. Nature genetics, 43(2), 101-108.

Atkinson, R. G., Cipriani, G., Whittaker, D. J., & Gardner, R. C. (1997). The allopolyploid origin of kiwifruit, Actinidia deliciosa (Actinidiaceae). Plant Systematics and Evolution, 205(1-2), 111-124.

Barone, A., Chiusano, M. L., Ercolano, M. R., Giuliano, G., Grandillo, S., & Frusciante, L. (2008). Structural and functional genomics of tomato. International journal of plant genomics, 2008.

Barrett, C. F., McKain, M. R., Sinn, B. T., Ge, X. J., Zhang, Y., Antonelli, A., & Bacon, C. D. (2019). Ancient polyploidy and genome evolution in palms. Genome biology and evolution, 11(5), 1501-1511. 

Besnard, G., Garcia-Verdugo, C., Rubio de Casas, R., Treier, U. A., Galland, N., & Vargas, P. (2008). Polyploidy in the olive complex (Olea europaea): evidence from flow cytometry and nuclear microsatellite analyses. Annals of Botany, 101(1), 25-30.

Castro, J. P., Moraes, A. P., Chase, M. W., Santos, A., Batista, F. R., & Felix, L. P. (2020). Karyotype characterization and evolution of chromosome number in Cactaceae with special emphasis on subfamily Cactoideae. Acta Botanica Brasilica, 34(1), 135-148.

Chanderbali, A. S., Albert, V. A., Ashworth, V. E., Clegg, M. T., Litz, R. E., Soltis, D. E., & Soltis, P. S. (2008). Persea americana (avocado): bringing ancient flowers to fruit in the genomics era. BioEssays, 30(4), 386-396.

Clarkson, J. J., Dodsworth, S., & Chase, M. W. (2017). Time-calibrated phylogenetic trees establish a lag between polyploidisation and diversification in Nicotiana (Solanaceae). Plant Systematics and Evolution, 303(8), 1001-1012.

Escudero, M., Balao, F., Martín‐Bravo, S., Valente, L., & Valcárcel, V. (2018). Is the diversification of Mediterranean Basin plant lineages coupled to karyotypic changes?. Plant Biology, 20, 166-175.

Illa, E., Sargent, D. J., Girona, E. L., Bushakra, J., Cestaro, A., Crowhurst, R., ... & Gardiner, S. (2011). Comparative analysis of rosaceous genomes and the reconstruction of a putative ancestral genome for the family. BMC Evolutionary biology, 11(1), 1-13.

Kim, Y. M., Kim, S., Koo, N., Shin, A. Y., Yeom, S. I., Seo, E., ... & Jang, I. (2017). Genome analysis of Hibiscus syriacus provides insights of polyploidization and indeterminate flowering in woody plants. Dna Research, 24(1), 71-80.

Lattier, J. D., Chen, H., & Contreras, R. N. (2019). Variation in genome size, ploidy, stomata, and rDNA signals in Althea. Journal of the American Society for Horticultural Science, 144(2), 130-140.

Leal, F., Loureiro, J., Rodriguez, E., Pais, M. S., Santos, C., & Pinto-Carnide, O. (2006). Nuclear DNA content of Vitis vinifera cultivars and ploidy level analyses of somatic embryo-derived plants obtained from anther culture. Plant Cell Reports, 25(9), 978-985.

Martin, G., Carreel, F., Coriton, O., Hervouet, C., Cardi, C., Derouault, P., ... & Labadie, K. (2017). Evolution of the banana genome (Musa acuminata) is impacted by large chromosomal translocations. Molecular biology and evolution, 34(9), 2140-2152.

Matuszak-Renger, S., Paule, J., Heller, S., Leme, E. M., Steinbeisser, G. M., Barfuss, M. H., & Zizka, G. (2018). Phylogenetic relationships among Ananas and related taxa (Bromelioideae, Bromeliaceae) based on nuclear, plastid and AFLP data. Plant Systematics and Evolution, 304(7), 841-851.

Mayrose, I., Barker, M. S., & Otto, S. P. (2010). Probabilistic models of chromosome number evolution and the inference of polyploidy. Systematic biology, 59(2), 132-144.

Melo, C. A. F., Martins, M. I. G., Oliveira, M. B. M., Benko-Iseppon, A. M., & Carvalho, R. (2011). Karyotype analysis for diploid and polyploid species of the Solanum L. Plant Systematics and Evolution, 293(1-4), 227-235.

Peruzzi, L., Leitch, I. J., & Caparelli, K. F. (2009). Chromosome diversity and evolution in Liliaceae. Annals of Botany, 103(3), 459-475.

Roullier, C., Duputié, A., Wennekes, P., Benoit, L., Bringas, V. M. F., Rossel, G., ... & Lebot, V. (2013). Disentangling the origins of cultivated sweet potato (Ipomoea batatas (L.) Lam.). PLoS One, 8(5), e62707.

Sousa, W. R. D. N., Lopes, A. C. D. A., Carvalho, R. D., Gomes, R. L. F., & Peron, A. P. (2015). Karyotypic characterization of Capsicum sp. accessions. Acta Scientiarum. Agronomy, 37(2), 147-153.

Spooner, D., Rojas, P., Bonierbale, M., Mueller, L. A., Srivastav, M., Senalik, D., & Simon, P. (2013). Molecular phylogeny of Daucus (Apiaceae). Systematic Botany, 38(3), 850-857.

Stein, J. C., Yu, Y., Copetti, D., Zwickl, D. J., Zhang, L., Zhang, C., ... & Wei, S. (2018). Genomes of 13 domesticated and wild rice relatives highlight genetic conservation, turnover and innovation across the genus Oryza. Nature genetics, 50(2), 285-296.

Tiffin, P., & Gaut, B. S. (2001). Sequence diversity in the tetraploid Zea perennis and the closely related diploid Z. diploperennis: insights from four nuclear loci. Genetics, 158(1), 401-412.

Wang, J., Sun, P., Li, Y., Liu, Y., Yang, N., Yu, J., ... & Ge, D. (2018). An overlooked paleotetraploidization in Cucurbitaceae. Molecular biology and evolution, 35(1), 16-26.

Watanabe, K. (2015). Potato genetics, genomics, and applications. Breeding science, 65(1), 53-68.

Wendel, J. F. (2015). The wondrous cycles of polyploidy in plants. American journal of botany, 102(11), 1753-1756.

Williams, W. M., Verry, I. M., Ansari, H. A., Hussain, S. W., Ullah, I., & Ellison, N. W. (2019). A Eurasia-wide polyploid species complex involving 6x Trifolium ambiguum, 2x T. occidentale and 4x T. repens produces interspecific hybrids with significance for clover breeding. BMC plant biology, 19(1), 438.

Wu, G. A., Terol, J., Ibanez, V., López-García, A., Pérez-Román, E., Borredá, C., ... & Curk, F. (2018). Genomics of the origin and evolution of Citrus. Nature, 554(7692), 311-316.

Yeasmin, L., Ali, M. N., Gantait, S., & Chakraborty, S. (2015). Bamboo: an overview on its genetic diversity and characterization. 3 Biotech, 5(1), 1-11.

Zhuang, W., Chen, H., Yang, M., Wang, J., Pandey, M. K., Zhang, C., ... & Garg, V. (2019). The genome of cultivated peanut provides insight into legume karyotypes, polyploid evolution and crop domestication. Nature genetics, 51(5), 865-876.
